libnavigation-core/src/main/java/com/mapbox/navigation/core/navigator/NavigatorMapper.kt

```diff
@@ -654,3 +665,12 @@ internal fun NavigationStatus.prepareSpeedLimit(): SpeedLimit? {
         )
     }
 }
+
+internal sealed class MappedRoadObject {
+    internal data class SingleObject(val roadObject: RoadObject) : MappedRoadObject()
+    internal data class EntranceAndExit(
```

Comment:
Can these be `private` instead of `internal`?

Reply:
no, can't be private. 
All mapping functions inside `NavigatorMapper` are `separate` private functions. So, if we make the classes ^^ `private` they will be unavailable for functions.

Reply:
`MappedRoadObject` can be `private` though.

Reply:
ah, right, I checked with `SingleObject ` and `EntranceAndExit `. but only parent `MappedRoadObject ` can be private